docs/abbr.md

```diff
@@ -152,3 +152,4 @@ Note that there are always exceptions, especially when we try to comply with the
 |                  | activation                          | actn           |                                                  |
 |                  |                                     |                |                                                  |
 | **Text**         | string                              | s              |                                                  |
+|                  | token                               | tok            |                                                  |
```

Comment:
did you mean this to be 'n_tok' instead of 'tok'? I could be mistaken ...

Reply:
No `tok` is correct here

Reply:
No, there is an entry on line 30 defining the prefix `n_`.  So this like defines `tok` for `token` and `n_tok` is the number of tokens.